### DISCUSSION TITLE
Flarum容器重启后网站的扩展消失

### DISCUSSION THREAD

**Discussion by anewinstance** [1Panel-dev/1Panel#6092] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

Flarum容器重启后网站的扩展消失

### 重现步骤

安装flarum，添加一些扩展，重启容器

### 期待的正确结果

添加的扩展保保留

**Comment by wanghe-fit2cloud** [comment]
在 1Panel 应用商店中安装的 Flarum 将 `./data` 目录设置为持久化目录。如果遇到扩展丢失的问题，可能是由于扩展目录未被正确持久化所致。建议查阅 Flarum 官方文档，了解是否有相关说明。
> 也可以进入 `容器` - `终端`，确定一下扩展目录。后面在安装应用时将扩展目录映射到宿主机。
![image](https://github.com/user-attachments/assets/698a0320-89de-49e4-8be1-9f483667e2cf)

**Comment by ShoKo-oo** [comment]
Flarum 的扩展目录似乎不在./data中 而是通过composer管理扩展下载保存在vendor中还是extensions 中来着，不过我还不清楚如何优雅的解决这个问题
